src/Common/src/Interop/Unix/System.Native/Interop.Connect.cs

```diff
@@ -10,6 +10,23 @@ internal static partial class Interop
     internal static partial class Sys
     {
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_Connect")]
-        internal static extern unsafe Error Connect(int socket, byte* socketAddress, int socketAddressLen);
+        private static extern unsafe Error DangerousConnect(int socket, byte* socketAddress, int socketAddressLen);
```

Comment:
I think in other places we've used a suffix like Private.  But since the naming here is likely temporary anyway, seems fine as is.